projects/plugins/jetpack/_inc/lib/class.jetpack-iframe-embed.php

```diff
@@ -24,35 +33,43 @@ static function init() {
 		add_filter( 'shortcode_atts_audio', array( 'Jetpack_Iframe_Embed', 'disable_autoplay' ) );
 
 		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
-			wp_enqueue_script( 'jetpack-iframe-embed', WPMU_PLUGIN_URL . '/jetpack-iframe-embed/jetpack-iframe-embed.js', array( 'jquery' ) );
+			wp_enqueue_script(
+				'jetpack-iframe-embed',
+				WPMU_PLUGIN_URL . '/jetpack-iframe-embed/jetpack-iframe-embed.js',
+				array( 'jquery' ),
+				JETPACK__VERSION,
```

Comment:
On WP.com right now, `JETPACK__VERSION` is always `wpcom`. What about using the same `$ver` as below in both places?

Reply:
I'm okay with that. b77390ea13870a612d0037a0994315869d98f838